06-package-management.Rmd

```diff
@@ -98,11 +98,10 @@ In your scientific notebook, you'll need to add two items.
 <details> <summary>**R version of the exercise**</summary>
 
 1. In your Rmd file, add a chunk in the very end that looks like this:
-`````
 ```{r}
 sessionInfo()
 ```
-`````
```

Comment:
In the html of this course, this code seems to be generating not only the R-styled snippet for the sessionInfo command but also an entire example output of sessionInfo. I'm not sure but I don't think the intention is to add this sessionInfo to the Rmd file?  I am guessing that the 5 back ticks are not having the intended effect here?

Reply:
I separated the output and input now